### DISCUSSION TITLE
NovelHi.com support possible?

### DISCUSSION THREAD

**Discussion by DaoDaoDao12** [lncrawl/lightnovel-crawler#1289] [Ideas]
Is support for NovelHi.com possible in the future?

It features extremely high quality MTL translations, perhaps the best I've ever read.

**Comment by dipu-bd** [comment]
https://github.com/dipu-bd/lightnovel-crawler/issues/1263

A issue request is already there. Wait for us to work on it.

**Reply by torkita** [reply]
> #1263
> 
> A issue request is already there. Wait for us to work on it.

I'm still waiting, any updates on it?
